src/rust/ensogl/src/display/symbol/gpu.rs

```diff
@@ -348,6 +348,18 @@ impl Symbol {
                 let count          = self.surface.point_scope().size()    as i32;
                 let instance_count = self.surface.instance_scope().size() as i32;
 
+                // Check if we are ready to render. If we don't assert here we wil only get a warning
+                // that won't tell us where things went wrong.
+                {
+                    let framebuffer_status = context.check_framebuffer_status(Context::FRAMEBUFFER);
+                    debug_assert_eq!(
+                        framebuffer_status,
+                        Context::FRAMEBUFFER_COMPLETE,
+                        "Framebuffer does not have status FRAMEBUFFER_COMPLETE, but {}.",
```

Comment:
A very minor thing. I would make the error sentence a little bit more standard (as in other implementations). I would use here `"Framebuffer incomplete (status: {})."` instead :) This is shorter and provides the same information.